src/pb_tensor.cc

```diff
@@ -305,11 +339,30 @@ PbTensor::Memory()
 
 #ifdef TRITON_PB_STUB
 std::shared_ptr<PbTensor>
-PbTensor::FromDLPack(const std::string& name, const py::capsule& dlpack_tensor)
+PbTensor::FromDLPack(const std::string& name, const py::object& tensor)
 {
   if (name == "") {
     throw PythonBackendException("Tensor name cannot be an empty string.");
   }
+  if(py::isinstance<py::capsule>(tensor)){
+    return FromDLPackCapsule(name, tensor);
+  } else if (py::hasattr(tensor, "__dlpack__")) {
+    // Python backend does not support async executions on PbTensor objects,
```

Comment:
I was wondering if this is documented somewhere? Just wanted to read more about it but couldn't find more information in README file.

Reply:
@Tabrizian we've discussed offline that async execution is not supported. Do we have it explicitly stated in docs?